### ISSUE TITLE
Known issue in the cronjob tutorial

### ISSUE THREAD

**Issue by mengqiy** [kubernetes-sigs/kubebuilder#861]
There is a known issue in the latest cronjob tutorial.
You may see errors complaining about `.metadata.creationTimestamp` under spec.

Here's the workaround.
Add the following into file `config/crd/kustomization.yaml`

```
patchesJson6902:
- target:
    group: apiextensions.k8s.io
    version: v1beta1
    kind: CustomResourceDefinition
    name: cronjobs.batch.tutorial.kubebuilder.io
  path: patches/validation_schema_cronjobs.yaml
```

And then create file `config/crd/patches/validation_schema_cronjobs.yaml`

```
- op: remove
  path: /spec/validation/openAPIV3Schema/properties/spec/properties/jobTemplate/properties/metadata/properties/creationTimestamp

- op: remove
  path: /spec/validation/openAPIV3Schema/properties/spec/properties/jobTemplate/properties/spec/properties/template/properties/metadata/properties/creationTimestamp
```

/kind bug

**Comment by mengqiy** [comment]
@droot is trying to fix it in controller-tools.

**Comment by droot** [comment]
This is fixed in `2.0.0-beta.0` release where we dropped schema generation for `objectMeta`.
